tachyon/math/finite_fields/binary_fields/binary_fields_unittest.cc

```diff
@@ -1,12 +1,12 @@
-#include "gtest/gtest.h"
-
 #include "tachyon/math/finite_fields/binary_fields/binary_fields.h"
 
+#include "gtest/gtest.h"
+
 namespace tachyon::math {
 
 namespace {
 template <typename BinaryField>
-class BinaryFieldTest : public testing::Test {
+class BinaryFieldsTest : public testing::Test {
```

Comment:
Did you intend to rename? Also the file name right?

Reply:
Oh yes, this is because the file name is `binary_fields_unittest.cc`.